pages/basePage.ts

```diff
@@ -0,0 +1,15 @@
+import { Page } from "@playwright/test";
+
+export class BasePage {
+  readonly page: Page;
+  readonly maxUserNameLength = 14;
```

Comment:
these variables wiil not suitable for all pages we will use. It's BasePage and should contains only base/generic variables, methods.